components/cam/src/physics/cam/micro_p3.F90

```diff
@@ -1455,6 +1474,24 @@ SUBROUTINE p3_main(qc,nc,qr,nr,th_old,th,qv_old,qv,dt,qitot,qirim,nitot,birim,ss
           ! The rates are adjusted here (where necessary) such that the sum of the sinks of mass cannot
           ! be greater than the sum of the sources, thereby resulting in overdepletion.
 
+          ! map warm-phase process rates to cell-avg
```

Comment:
The "Make process rates cell-average" section should come before the "conservation of water" header and associated comment since it's really a different operation.

Reply:
oh yes, good point.  I'll move the comment, and add some more descriptive comments.